Add description of behavior to explainer (#42)
jackbsteinberg/std-toast: README.md

### The `<std-toast>` element

#### Behavior

The `<std-toast>` element provides a subtle, non-interruptive notification to the user.
The toast will appear at a customized time and position on the screen, 
and will typically contain a message and optionally action and dismiss buttons

> (though other markup in the element is supported).

Reviewer: ```suggestion
(though arbitrary markup in the element is supported).
```

Right?

Author: Yeah that's clearer

The contents will be announced to a screen reader

> (politely or assertively depending on `type` [see [attributes](####attributes)]),

Reviewer 2: ```suggestion
(politely or assertively depending on `type` [see [attributes](#attributes)]),
```

> and the optional action / dismiss buttons will appear next in the tabbing order.

Reviewer 2: ```suggestion
and the focusable contents of the toast, if any, will appear next in the tabbing order.
```

After a certain duration, the toast will timeout and hide itself,

> though this timeout will be suspended while the toast has focus or `mousehover`.

Reviewer 2: ```suggestion
though this timeout will be suspended while the toast has focus or the mouse is hovering over it.
```

(`mousehover` is not a defined term that I know of.)

Author: Should I change this to `mouseover` instead? I think that's a defined term

Reviewer 2: Yeah, that's reasonable, I think. Depends on whether you want to use the event terms or just general English descriptions.

Author: I changed my mind, yours sounds better, going with that
